### PR TITLE
Move ECS service module to template directory

### PR DESCRIPTION
**Pull request by likelymichael**
Passing local deploys and published to massdriver to test various deploys of various configs and changing configs, redeploys, decommissions, etc.

### CODE REVIEW

File: aws-ecs-service/src/README.md
Comment (chrisghill) [inline aws-ecs-service/src/README.md:1]: I think this should be up one directory, yeah?

**Review by chrisghill — APPROVED** [review]
This looks good, but the main test is whether or not it deploys properly.

**Comment by likelymichael** [comment]
> This looks good, but the main test is whether or not it deploys properly.

I should clarify in the details that it was doing all those deploys and reconfigures and decommissions from the platform, meaning it is deploying properly

File: aws-ecs-service/src/README.md
Comment (likelymichael) [reply aws-ecs-service/src/README.md:1]: Yes, you're right, it should. I'll move it

**Pull request merged**